common_utils/constants.go

```diff
@@ -0,0 +1,4 @@
+
+package common_utils
+
+const GNMI_WORK_PATH = "/etc/sonic/gnmi"
```

Comment:
>/etc/sonic/gnmi [](http://example.com/codeflow?start=24&length=15)

/etc is for config files. Working path may fit more under /var/run.

Reply:
Both host and gNMI container can access /etc/sonic. So gNMI server can create a patch file in this path, and notify host to run “config apply-patch" with this patch file.
Do you think we should create a working path under /var/run, and share this path between host and gNMI container?